src/utils.js

```diff
@@ -416,6 +417,82 @@ function getComponentsJson(logger, callback) {
     return deferred.promise.nodeify(callback);
 }
 
+function createStartUpProjects(gmeConfig, gmeAuth, storage, logger, url) {
+    var deferred = Q.defer(),
+        WorkerRequest = require('./server/worker/workerrequests'),
+        worker = new WorkerRequest(logger.fork('worker'), gmeConfig, url),
+        configArray = JSON.parse(JSON.stringify(gmeConfig.seedProjects.createAtStartup)),
+        promises = [],
+        creators = [],
+        tokens = [],
+        createdProjects = [];
+
+
+    configArray.forEach(function (projectInfo) {
+        projectInfo.creatorId = projectInfo.creatorId || gmeConfig.authentication.admin;
+        projectInfo.ownerId = projectInfo.ownerId || projectInfo.creatorId;
+
+        if (creators.indexOf(projectInfo.creatorId) === -1) {
+            creators.push(projectInfo.creatorId);
+            promises.push(gmeAuth.generateJWTokenForAuthenticatedUser(projectInfo.creatorId));
+        }
+    });
+
+    Q.all(promises)
+        .then(function (tokens_) {
+            var promises = [];
+
+            tokens = tokens_;
+            creators.forEach(function (owner) {
+                promises.push(storage.getProjects({user: owner}));
```

Comment:
Make sure to write a test that catches this.. It still uses the guest account - it should be username..

Reply:
cr.p